File: solutions/beecrowd/3040/3040.py
def is_valid_tree(a, d, g):
    return a >= 200 and a <= 300 and d >= 50 and g >= 150


n = input()

for _ in range(int(n)):
    line = input()
    a, d, g = map(int, line.split())
    print('Sim' if is_valid_tree(a, d, g) else 'Nao')

File: solutions/beecrowd/3040/3040.cpp
#include <cstdint>
#include <iostream>

bool is_valid_tree(int16_t a, int16_t d, int16_t g) {
    return a >= 200 && a <= 300 && d >= 50 && g >= 150;
}

int main() {
    int16_t n, a, d, g;

    std::cin >> n;

    for (int16_t i = 0; i < n; i++) {
        std::cin >> a >> d >> g;
        std::cout << (is_valid_tree(a, d, g) ? "Sim" : "Nao") << std::endl;
    }

    return 0;
}
